### PR TITLE
Add a tutorial about a new experimental onClick and onDrag modifiers

### PR DESCRIPTION
**Pull request by eymar**

### CODE REVIEW

**Review by dima-avdeev-jb — APPROVED** [review]

**Comment by igordmn** [comment]
We need to remove tutorial about mouseClickable and "Advanced click events processing", because new API covers cases in those tutorials

File: tutorials/Mouse_Events/README.md [hunk 1]
```diff
@@ -415,3 +415,236 @@ fun main() = singleWindowApplication {
...
+import androidx.compose.ui.input.pointer.isAltPressed
+import androidx.compose.ui.input.pointer.isShiftPressed
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.singleWindowApplication
+
+@OptIn(ExperimentalFoundationApi::class, ExperimentalAnimationApi::class)
+fun main() {
+    singleWindowApplication {
+        Column {
+            var topBoxText by remember { mutableStateOf("Click me") }
```
Comment (igordmn) [inline tutorials/Mouse_Events/README.md:457]: Better to write more info which action should the user perform. Something like: "Click me with LMB or Shift+LMB" (and add `, textAlign = TextAlign.Center` to the text)

**Review by igordmn — COMMENTED** [review]
Thanks!

File: tutorials/Mouse_Events/README.md [hunk 2]
```diff
@@ -415,3 +415,236 @@ fun main() = singleWindowApplication {
...
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.input.pointer.PointerButton
+import androidx.compose.ui.input.pointer.isAltPressed
+import androidx.compose.ui.input.pointer.isShiftPressed
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.singleWindowApplication
+
+@OptIn(ExperimentalFoundationApi::class, ExperimentalAnimationApi::class)
+fun main() {
+    singleWindowApplication {
```
[the remark this answers is not in the document]
Comment (eymar) [reply tutorials/Mouse_Events/README.md:455]: I didn't notice it. I'll follow the convention.

**Review by igordmn — APPROVED** [review]

**Pull request merged**